tests/classifiers/simple_classifier_tests.py

```diff
@@ -24,18 +24,39 @@
 warnings.filterwarnings("ignore")
 
 from sklearn import datasets
+from sklearn.feature_extraction.text import TfidfVectorizer
 from sklearn.metrics import accuracy_score
 from sklearn.model_selection import train_test_split
 
 from simple_learn.classifiers import SimpleClassifier
 
 
 class TestSimpleClassifier(unittest.TestCase):
+    """
+    Tests for the SimpleClassifier Class
+    """
+
     def test_init(self):
+        """
+        Test initialization of SimpleClassifier
+
+        Expected
+        -----------------
+        name : "Empty Model"
+        """
         clf = SimpleClassifier()
         self.assertEqual(clf.name, "Empty Model")
 
     def test_wine(self):
+        """
+        Test SimpleClassifier against sklearn wine dataset
+
+        Expected
+        -----------------
+        model : Not None
+        training accuracy : > 0.0
```

Comment:
Awesome work @sh-biswas, I really appreciate you adding documentation as well! Nothing wrong with your PR, but I forgot to change the expected values of training accuracy from 0.0. Would you mind changing the expect value of all the training accuracies to 0.95 before we merge this?

Reply:
Fixed!